src/components/tier.jsx

```diff
@@ -47,18 +49,28 @@ class Tier extends React.Component {
 
   render() {
 
-    let items = this.state.items;
+    var costText = this.state.tierCost;
 
-    let itemsArray = [];
+    var items = this.state.items;
 
-    for (let key in items) {
-      let itemId = items[key];
+    var itemsArray = [];
+
+    for (var key in items) {
+      var itemId = items[key];
       itemsArray.push(<Item key={itemId} itemId={itemId}/>)
     }
 
+    if (this.state.tierType === "1") {
+      costText = "PAY $1 TO UNLOCK!";
+    } else if (this.state.tierType === "2") {
+      costText = "PAY $" + this.state.tierCost + " OR MORE TO UNLOCK!";
+    } else if (this.state.tierType === "3") {
+      costText = "PAY $" + this.state.tierCost + " OR MORE TO ALSO UNLOCK!";
```

Comment:
String literals would help make this more readable. Also, I think we're meant to use single quotes typically unless double quotes are expected.